requirements/requirements.txt

```diff
@@ -39,7 +39,7 @@ pytz==2014.10
 PyYAML==3.10
 raven==3.1.17
 requests==2.0.0
-south==0.7.3
+south==1.0.2
```

Comment:
Ran through their releases: http://south.readthedocs.org/en/latest/releasenotes/index.html

There aren't any incompatible changes, although we will want to move our current migrations to south_migrations folders before upgrading to django 1.7